pkg/runctl/run_manager_test.go

```diff
@@ -1907,18 +1980,31 @@ func Test__runManager__Log_Elasticsearch(t *testing.T) {
 }
 
 type testHelper1 struct {
-	t            *testing.T
-	ctx          context.Context
-	namespace1   string
-	pipelineRun1 string
+	t             *testing.T
+	ctx           context.Context
+	namespace1    string
+	pipelineRun1  string
+	runNamespace1 string
+}
+
+func newTestHelperNoRunNamespace(t *testing.T) *testHelper1 {
+	h := &testHelper1{
+		t:             t,
+		ctx:           context.Background(),
+		namespace1:    "namespace1",
+		pipelineRun1:  "pipelinerun1",
+		runNamespace1: "",
+	}
+	return h
 }
```

Comment:
In my opinion this function is not justified. A test should simply use `newTestHelper1` and set field runNamespace1 of the returned value to `""`.

Reply:
See https://github.com/SAP/stewardci-core/pull/345/commits/25512b11ea2424ea1b3d857a7bbc8bed4a3ce067